types/Types.tsx

```diff
@@ -3,3 +3,10 @@ export interface TextboxDataType {
     answer: string | undefined;
     candidateId: number;
 }
+
+export interface approvaltextboxprop {
```

Comment:
shouldn't answerId, candidateId and points be numbers?

Reply:
The type of points is optional, depending on how we treat our data. Du you strip it when saving to local storage?
CandidateId is a string in the inspera dataset, but I dont know what type you save it as in local storage?
The answerid has an underscore, so it has to be a string.

Reply:
I thought since we are gonna have to change the code when applying the local storage-code (like removing the testdata) anyway, we can possibly adjust the types depending on how the data turns out 🤷

Reply:
When saving points to localstorage, selectedOption.value is a number (but I havent managed to save it yet)

Reply:
Haha, okey, do you want me to change it to type number?